src/api/Gemfile

```diff
@@ -188,11 +188,11 @@ group :assets do
   # assets for jQuery and jQuery-ujs
   gem 'jquery-rails'
   # assets for jQuery-ui
-  gem 'jquery-ui-rails', '~> 4.2.1' # version 5 needs henne's new webui
-  # assets for the bootstrap front-end framework. Used by the bratwurst theme
-  # gem 'bootstrap-sass-rails'
+  gem 'jquery-ui-rails', '~> 4.2.1'
+  # assets for the bootstrap front-end framework
+  gem 'bootstrap'
```

Comment:
Nope, we talked about using webpack but decided against it for now...

Reply:
Please no webpack in my life. 👍

Reply:
@hennevogel 

> Nope, we talked about using webpack but decided against it for now...

but I am speaking about adding bootstrap files directly instead of using the gem. I think this is not related with webpack :confused:

Reply:
It seems I am not the only onw who has this question today: https://stackoverflow.com/questions/50870738/why-use-ruby-gem-for-bootstrap-when-you-can-just-one-line-include-bootstrap-in-a :rofl:

Reply:
I can tell you why: to theme bootstrap with sass.

Reply:
> to theme bootstrap with sass

Why can you not do this without the gem? :thinking: 

> privacy

why?

Reply:
> why?

Because every request to an OBS page would (potentially) make a request to whatever CDN you use? You think they offer this service out of the goodness of their :heart:? :smiley:

Reply:
Sass requires to be compiled locally, externally, in CDN, CSS is used and can't be recompiled to feature changes that you (or 3rd party theme having person wanting to use OBS for themselves with their corporate/community theme) want to do

Reply:
ok, thanks for the explanation :+1: